docs/dg/architecture.md

```diff
@@ -27,6 +27,7 @@
 ## Git
 
 [`Git`](https://github.com/reposense/RepoSense/blob/master/src/main/java/reposense/git) package contains the wrapper classes for respective *git* commands.
+Note that when constructing new commands containing path arguments, use `StringsUtil::addQuotesForFilePath` method to safely convert a Java string into an equivalent Bash/CMD argument.
```

Comment:
Should move to the end of this section (Git) and use `<box type="info" seamless>` so that it looks nicer on the DG.

Reply:
Have changed it to use markbind's box. Seems like there needs to be an empty line after the first `<box>` tag to properly parse code wrapped in " ` ".